core/templates/pages/library-page/library-page.controller.spec.ts

```diff
@@ -40,6 +42,17 @@ describe('Library controller', function() {
     var scope, ctrl, $httpBackend;
     var $componentController;
 
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        providers: [KeyboardShortcutService]
+      });
+    });
+
+    beforeEach(angular.mock.module('oppia', function($provide) {
+      $provide.value('KeyboardShortcutService',
```

Comment:
Break line after `(` [Following `There should be a line break after `(` if the params are spread in multi-lines)